pkg/otlp/metrics/dimensions.go

```diff
@@ -78,11 +78,18 @@ func (d *Dimensions) OriginProductDetail() OriginProductDetail {
 }
 
 // getTags maps an attributeMap into a slice of Datadog tags
-func getTags(labels pcommon.Map) []string {
+func getTags(labels pcommon.Map, splat bool) []string {
 	tags := make([]string, 0, labels.Len())
 	labels.Range(func(key string, value pcommon.Value) bool {
-		v := value.AsString()
-		tags = append(tags, utils.FormatKeyValueTag(key, v))
+		if splat && value.Type() == pcommon.ValueTypeSlice {
```

Comment:
Are there slices of `X` type that we should be handling differently/ testing for ?

Reply:
I don't think this will happen ever because of open-telemetry/opentelemetry-specification/pull/3858 but I implemented it just in case, nice suggestion :)